packages/eslint-config-sheriff/tsup.config.js

```diff
@@ -8,15 +8,5 @@ export default defineConfig({
   clean: true,
   noExternal: ['@sherifforg/constants', '@sherifforg/types'],
   skipNodeModulesBundle: true,
-  dts: {
-    entry: 'src/index.ts',
-    resolve: true,
-  },
-  banner: ({ format }) => {
```

Comment:
uhm, this code was necessary

Reply:
This injects a global require symbol, which isn't needed for a package authored as ESM and which doesn't bundle its dependencies.
It _just_ hides problems with the bundling config. I've manually verified it works as expected.